pkg/target/target.go

```diff
@@ -337,12 +344,13 @@ func (h *K8sValidationTarget) MatchSchema() apiextensions.JSONSchemaProps {
 					},
 				},
 			},
-			"namespaces":         wildcardNSList,
-			"excludedNamespaces": wildcardNSList,
-			"labelSelector":      labelSelectorSchema,
-			"namespaceSelector":  labelSelectorSchema,
+			"namespaces":         propsWithDescription(wildcardNSList, "namespaces is a list of namespace names. If defined, a constraint will only apply to resources in a listed namespace."),
```

Comment:
"list of names" is no longer accurate, since we accept glob format as well. This also applies for `excludedNamespaces`. We should also add that it is ignored for cluster scoped resources.

Reply:
I've updated these with what's currently in the `howto.md`, which includes info on globs.  Have a look and LMK if you don't think it makes sense.